product_multi_barcode_constraint_per_company/tests/test_product_barcode_constraint.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import types
+
+from psycopg2 import IntegrityError
+
+from odoo.tests.common import tagged
+from odoo.tools.misc import mute_logger
+
+from odoo.addons.product_barcode_constraint_per_company.tests.test_module import (
+    TestModule,
+)
+
+from .. import void
+
+
+@tagged("post_install", "-at_install")
+class TestProductBarcodeConstraintInherit(TestModule):
```

Comment:
If you want to inherit the setup from your other tests, but not the tests themselves, you should separate the setup function into its own class so you can inherit from it separately.

Here's an example: https://github.com/OCA/stock-logistics-workflow/blob/14.0/stock_picking_auto_create_lot/tests/test_stock_picking_auto_create_lot.py#L10

Reply:
Thanks for the advice, I will definitely use it.